src/libsyntax/ext/tt/quoted.rs

```diff
@@ -0,0 +1,228 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
```

Comment:
Why make this module here? I'd have thought TTs were much more general than just quasi quoting